### PR TITLE
docs: add FAST to showcase

### PR DESCRIPTION
**Pull request by huynhicode**
### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Run `yarn start`, view the changes on `http://localhost:3000/showcase` or `http://localhost:3000/showcase?name=fast`

## Related PRs

None

---

Screenshot:

Screen Shot 2022-04-01 at 2 28 44 PM

### CODE REVIEW

**Comment by Josh-Cena** [comment]
@huynhicode Please remember to sign the CLA. PRs without CLA will be closed after two weeks. Thanks!

**Review by slorber — CHANGES_REQUESTED** [review]
thanks, 

CLA signature is required

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -549,6 +549,14 @@ const Users: User[] = [
     source: 'https://github.com/DigiPie/kaya-folio',
     tags: ['opensource', 'personal'],
   },
+  {
+    title: 'FAST',
+    description: 'The adaptive interface system for modern web experiences.',
+    preview: require('./showcase/fast.png'),
+    website: 'https://www.fast.design/',
+    source: 'https://github.com/microsoft/fast/tree/master/sites/website',
+    tags: ['opensource', 'product', 'design', 'large'],
```
Comment (slorber) [inline website/src/data/users.tsx:558]: doc is not particularly large + the design is not really related to Docusaurus itself

**Review by Josh-Cena — APPROVED** [review]
Thanks 👍 We can wait for the CLA as long as there's communication

**Comment by Josh-Cena** [comment]
@huynhicode Any news?

**Comment by Josh-Cena** [comment]
@huynhicode Hi, it's been another two weeks, any news?

I can appreciate how complicated these processes in big orgs can get, so don't get too stressed about it

**Comment by slorber** [comment]
This CLA friction is too annoying for site showcase submissions 😅 

Trying to see if I can get approval to implement a simpler workflow where site owners would only have to comment on a GitHub issue, requesting to add their site to the showcase.

As long as it's us committing, it shouldn't require site owners to sign the CLA 🤷‍♂️ 

Related to https://github.com/facebook/docusaurus/issues/6882

**Comment by huynhicode** [comment]
Hi @Josh-Cena Thanks for your patience, still pending approval.

@slorber That sounds like a great solution.

**Comment by Josh-Cena** [comment]
@huynhicode Is it fine if we close this PR and I send a PR myself, crediting you as co-author?

**Comment by Josh-Cena** [comment]
Thanks @EisenbergEffect I've sent https://github.com/facebook/docusaurus/pull/7384. Please take a look.

**Pull request closed without merging**